packages/main/src/plugin/image-registry.ts

```diff
@@ -650,8 +652,14 @@ export class ImageRegistry {
     return this.getManifestFromURL(manifestURL, imageData, token);
   }
 
-  async getAuthInfo(serviceUrl: string): Promise<{ authUrl: string; scheme: string }> {
+  async getAuthInfo(serviceUrl: string, ignoreInvalidCert?: boolean): Promise<{ authUrl: string; scheme: string }> {
     let registryUrl: string;
+    const options = this.getOptions();
+
+    // If ignoreInvalidCertificate was passed in, ignore the certificate
+    if (ignoreInvalidCert && options.https) {
```

Comment:
I'm wondering if the option need to be `ignoreInvalidCert` or `rejectUnauthorized` as the parameter already exist in tls
And in podman it's named `registries.insecure` so we end up with 3 properties for the same purpose.

Either rename to `rejectUnauthorized`  or `insecure` ?

thoughts ?

Reply:
Hmm, insecure would make the most sense as we will be ignoring the certificate and we do not support `http` since docker / podman does not support it.

I'll rename it!